src/HOCON/Impl/HoconPath.cs

```diff
@@ -16,7 +16,30 @@ namespace Hocon
     public sealed class HoconPath:List<string>, IEquatable<HoconPath>
     {
         public bool IsEmpty => Count == 0;
-        public string Value => string.Join(".", this);
+
+        public string Value
+        {
+            get
+            {
+                if (IsEmpty)
+                    return string.Empty;
+
+                List<string> pathSegments = new List<string>();
+                foreach (string subPath in this)
+                {
+                    // #todo escape newline chars in path?
+                    if (subPath.NeedTripleQuotes())
```

Comment:
This isn't needed, triple quotes in paths are rejected during creation, as they are illegal by requirement.

Reply:
Sorry, my code did not illustrate my concern properly. The problem I have is that a key is technically allowed to contain `\n` or other control characters. This creates a problem when you need to reference it in a path, as we can't triple quote it. I guess the solution is to escape all the offending characters automatically, which I was too lazy to implement.

To be clear, my code above does not fully address the concern above. Any thoughts?

Reply:
well, triple quotes in field keys are not allowed in the HOCON specification, and it is enforced when the key is created/parsed, so there is no need to check for a triple quote there.